### ISSUE TITLE
prefer-get false positive

### ISSUE THREAD

**Issue by fixpunkt** [wix/eslint-plugin-lodash#33]
For the following code snippet (where _mailing is a variable on the module scope):

      if (_mailing && _mailing.autosave && _mailing.autosave.modified) {
        return LocaleStore.formatMessage('autosave.navigatewarning')
      }

I get the following error:

    xxx/xxx.es6
    33:11  error  Prefer _.get or _.has over an '&&' chain  lodash/prefer-get

**Comment by fixpunkt** [comment]
I just noticed that this also happens for:

    const IS_DEVELOPMENT = process && process.env && process.env.NODE_ENV === 'development'

I've only started seeing this bug since updating from 1.1.1 to 1.2.1.

**Comment by ganimomer** [comment]
It doesn't look like these cases are false positives. In the first case, you could use
```js
  if (_.get(_mailing, ['autosave', 'modified']) {
    return LocaleStore.formatMessage('autosave.navigatewarning')
  }
```
and in the second case,
```js
const IS_DEVELOPMENT = _.get(process, ['env', 'NODE_ENV']) === 'development'
```
[In version 1.2.1](https://github.com/wix/eslint-plugin-lodash/blob/master/CHANGELOG.md#121---2016-02-25), we fixed a bug where `prefer-get` wasn't reporting where it should in ESLint 2, so that's why you got the errors for these examples.
Unless I'm missing something, these examples look like the ones in the [rule docs](https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/prefer-get.md).

**Comment by fixpunkt** [comment]
You're right of course, I wasn't thinking properly when I wrote the ticket.
